edge-inn-reach/src/main/resources/volaris/edge-inn-reach/global/jwt-token-helper.feature

```diff
@@ -0,0 +1,23 @@
+@ignore
+@parallel=false
+Feature: Edge inn reach JWT token helper
+
+  Background:
+    * url baseUrl
+
+    * callonce login admin
+    * def okapitokenUser = okapitoken
+    * def okapiTenantUser = testTenant
+    * def authToken1 = 'Basic MTczNWVkMmQtYTRhMS00YWE5LWFmYmYtOWUyZjc2YzNkMTZkOjU5YjliMDc1LTVmNjAtNGRlNi04YWIxLTQyODY2M2M4OGI4ZA=='
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'x-okapi-tenant': '#(okapiTenantUser)', 'Authorization' : '#(authToken1)', 'x-to-code': 'fli01', 'x-from-code': '69a3d', 'Accept': 'application/json'  }
+    * configure headers = headersUser
+
+  @GetJWTToken
+  Scenario: Authenticate credentials and get a new JWT token
+    * print 'Authenticate credentials and get a new JWT token'
+    Given url 'http://localhost:8081/innreach/v2/oauth2/token'
```

Comment:
This URL is hardcoded and will need to be changed if ran on vagrant box, please revisit this and change it to use a dynamic URL or an important comment needs to be added why the URL is using localhost:8081